### ISSUE TITLE
Add "toggle comment"

### ISSUE THREAD

**Issue by taiya** [pyzo/pyzo#528]
Most IDEs (with the exception of MATLAB, which does the same as Pyzo) allow commenting and uncommenting with a single shortcut (for example "CMD+/" in Textmate). In Pyzo you are forced to select two different shortcuts. It would be great if you could add a **Edit/Toggle comment** to resolve the problem?

**Comment by almarklein** [comment]
That actually makes a lot of sense :)

**Comment by almarklein** [comment]
Care for a PR?

**Comment by mullenkamp** [comment]
I would love to see this too! I'm switching over from Spyder.

**Comment by l33tlinuxh4x0r** [comment]
ctrl + 3
or 
ctrl + # 
would be preferred for me... I might look into this.

**Comment by l33tlinuxh4x0r** [comment]
I just made a pull request that fixes this bug.

**Comment by almarklein** [comment]
Closed by #718, thanks to @l33tlinuxh4x0r

**Comment by l33tlinuxh4x0r** [comment]
I have found an issue with my first revision of code. It literally toggles comments. For example:
```#line

#with

#comment

#would

#swap
```
would become
```line
#
with
#
comment
#
would
#
swap
```

I just cleaned up the code and implemented a fix I am going to lint it and then make another pull request.
